CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+### Added
+
+- `impl Clone` for {`CacheMe<_>`, `DefaultParseMode<_>`, `Throttle<_>`} ([#75][pr75])
```

Comment:
Is `<_>` needed? Previously we've been sticking to just `AutoSend`, for instance.

Reply:
I'm not sure. Do you think I should remove it?..

Reply:
Yes, I think it's just superfluous and makes the changelog harder to maintain.

Reply:
I don't think it's that important, but whatever